### PR TITLE
small tag replaced with div

### PR DESCRIPTION
**Pull request by yerramshilpa**

### CODE REVIEW

File: projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts [hunk 1]
```diff
@@ -9,15 +9,20 @@ import { FieldWrapper } from '@ngx-formly/core';
...
         *ngIf="to.label && !to.hideLabel"
         [attr.for]="id"
       >
-       {{ to.label }}
+        {{ to.label }}
       </label>
-      <small *ngIf="to.description" class="form-text text-muted" [innerHTML]="to.description"></small>
+      <div
+        *ngIf="to.description"
+        class="usa-label--description text-muted"
```
Comment (Bibek-Regmi) [inline projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts:22]: Do these 2 classes exist in uswds or are these sam-styles only classes?

Comment (yerramshilpa) [reply projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts:22]: its for sam-styles

Comment (Bibek-Regmi) [reply projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts:22]: We should not use sam-styles only classes within uswds projects. This project should work with just uswds. The request to update description might have been for sds repo and not this repo

Comment (yerramshilpa) [reply projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts]: usa-hint class updated for the description

**Review by Bibek-Regmi — APPROVED** [review]

**Pull request merged**